InternetStoreCHIReloaded/DAL/Migrations/20210817102603_AddedUserOrders.cs

```diff
@@ -0,0 +1,125 @@
+                {
+                    Id = table.Column<int>(type: "int", nullable: false)
+                        .Annotation("SqlServer:Identity", "1, 1"),
+                    Quantity = table.Column<int>(type: "int", nullable: false),
+                    ProductId = table.Column<int>(type: "int", nullable: false),
+                    OrderedDate = table.Column<DateTime>(type: "datetime2", nullable: false),
+                    OrderedPrice = table.Column<decimal>(type: "decimal(18,2)", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_OrderedProducts", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_OrderedProducts_Products_ProductId",
+                        column: x => x.ProductId,
+                        principalTable: "Products",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "Orders",
+                columns: table => new
+                {
+                    Id = table.Column<int>(type: "int", nullable: false)
+                        .Annotation("SqlServer:Identity", "1, 1"),
+                    CreationDate = table.Column<DateTime>(type: "datetime2", nullable: false),
+                    TotalSum = table.Column<decimal>(type: "decimal(18,2)", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Orders", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "OrderEntityOrderedProductEntity",
+                columns: table => new
+                {
+                    OrderItemsId = table.Column<int>(type: "int", nullable: false),
+                    OrdersId = table.Column<int>(type: "int", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_OrderEntityOrderedProductEntity", x => new { x.OrderItemsId, x.OrdersId });
+                    table.ForeignKey(
+                        name: "FK_OrderEntityOrderedProductEntity_OrderedProducts_OrderItemsId",
+                        column: x => x.OrderItemsId,
+                        principalTable: "OrderedProducts",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                    table.ForeignKey(
+                        name: "FK_OrderEntityOrderedProductEntity_Orders_OrdersId",
+                        column: x => x.OrdersId,
+                        principalTable: "Orders",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "OrderEntityUserEntity",
```

Comment:
Should relationship between Order and User be one-to-one?

Reply:
One to many*

Reply:
done